sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/SpecificParquetRecordReaderBase.java

```diff
@@ -152,6 +152,7 @@ protected void initialize(String path, List<String> columns) throws IOException
     Configuration config = new Configuration();
     config.setBoolean(SQLConf.PARQUET_BINARY_AS_STRING().key() , false);
     config.setBoolean(SQLConf.PARQUET_INT96_AS_TIMESTAMP().key(), false);
+    config.setBoolean(SQLConf.CASE_SENSITIVE().key(), false);
```

Comment:
Do we need to turn off `case-sensitivity` always?

Reply:
If this is recovered by somewhere else, could you add some comments before this line, please?

Reply:
This path is only used for testing, so I followed the above lines to just set the default value for this config.